lib/github.atd

```diff
@@ -630,14 +632,15 @@ type issue_comment_action = [
   | Created <json name="created">
   | Edited <json name="edited"> of body_changes
   | Deleted <json name="deleted">
-  | Unknown <json untyped> of (string * json option)
+  | Unknown of (string * json nullable)
```

Comment:
No change here on the ocaml/user side. This variant is not an enum because one case has an argument (`Edited of ...`) which prevents us from using `<json open_enum>`. The wrapping of unknown cases is done in the containing record below, using the annotation `<json adapter.ocaml="Github_json.Adapter.Issue_comment_event">`. `Github_json.Adapter.Issue_comment_event` is a module that provides json transformers.